app/src/main/java/in/testpress/testpress/authenticator/LoginActivity.java

```diff
@@ -522,6 +531,21 @@ protected void finishLogin() {
         finish();
     }
 
+    private void deleteOfflineVideosForDifferentAccount(String username) {
```

Comment:
What does this mean? This name does not make any sense. Please come up with new name.